src/requestor/routes/query.py

```diff
@@ -84,26 +92,42 @@ async def list_requests(
 @router.get("/request/user", status_code=HTTP_200_OK)
 async def list_user_requests(api_request: Request, auth=Depends(Auth)) -> dict:
     """
-    List the current user's requests.
+    List current user's requests.
 
     Use the "active" query parameter to get only those requests
     created by the user that are not in DRAFT or FINAL statuses.
     """
     # no authz checks because we assume the current user can read
     # their own requests.
     active = False
-    if "active" in api_request.query_params:
-        if api_request.query_params["active"]:
+    filter_dict = {k: [] for k in api_request.query_params if k != "active"}
```

Comment:
```suggestion
    filter_dict = {k: set() for k in api_request.query_params if k != "active"}
```
i'll suggest defaultdict(set) again, but this is ok too 🙂